path_generator:
  ros__parameters:
    #points stored as a flat list of doubles.
    # For each point: [x, y]
    #path_points: [1.2,0.0,1.2,1.2,0.0,1.2,0.0,0.0] #square
    #path_points: [1.0, 0.0, 1.5, 0.5, 1.5, -0.5, 1.0, 0.0] #cocktail glass normal
    #path_points: [1.5, 0.5, 1.5, -0.5, 0.0, 0.0, 1.0, 0.0] #cocktail glass alternate
    #path_points: [1.2, -1.2, 2.4, -0.6, 1.2, 0.0, 0.0, 0.0] #Hook 3101
    path_points: [0.5,0.0, 1.0,0.5, 1.5,0.5, 1.0,0.0] #Library Bend Forward
    path_points: [0.5,0.5, 1.0,0.5, 1.5,0.0, 1.0,0.0] #Library Bend Reverse

    #path_points: [4.8, 0.0] # Speed Validation (Yellow - Green)
path_control:
  ros__parameters:
    #Controller Gains
    kp_v: 0.2 #Linear velocity gain
    kp_w: 0.7 #Angular velocity gain
    #Tolerance Threshold
    goal_threshold: 0.02 
    #Maximum velocities
    max_v: 0.3 #Max velocity
    max_w: 0.3 #Max angular velocity
